PIDs:
  X:
    kp: 40.0
    kd: 20.0
    ki: 2.0
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -0.1
    integratorMax: 0.1
  Y:
    kp: -40.0
    kd: -20.0
    ki: -2.0
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -0.1
    integratorMax: 0.1
  Z:
    kp: 10000.0
    kd: 8000.0
    ki: 400
    minOutput: 10000.0
    maxOutput: 60000.0
    integratorMin: -800
    integratorMax: 800
  Yaw:
    kp: -70.0
    kd: -5.0
    ki: 0.0
    minOutput: -120.0
    maxOutput: 120.0
    integratorMin: -0.0
    integratorMax: 0.0
